snorkel/labeling/model/label_model.py

```diff
@@ -9,10 +9,9 @@
 
 from snorkel.analysis.utils import set_seed
 from snorkel.classification.utils import recursive_merge_dicts
-
-from .graph_utils import get_clique_tree
-from .lm_defaults import lm_default_config
-from .logger import Logger
+from snorkel.labeling.model.graph_utils import get_clique_tree
```

Comment:
Why? we're allowing `.` imports, just not `..` ones

Reply:
VSC debugger throws an error with these kinds of relative imports if for example you're trying to debug a sub-function called from tests.  If we're encouraging using VSC debugger, nice to have these be full import statements